sonar-plugin-api/src/main/java/org/sonar/api/rules/ActiveRule.java

```diff
@@ -120,6 +122,14 @@ public RulePriority getSeverity() {
   }
 
   /**
+   * For internal use
+   * @since 6.6
+   */
+  public RulePriority getOverriddenSeverity() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "RulePriority"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)